pkg/conversions/types_conversion_test.go

```diff
@@ -49,3 +49,44 @@ func TestConvert_CRD_PortMapping_To_SDK_PortMapping(t *testing.T) {
 		})
 	}
 }
+
+func TestConvert_CRD_Duration_To_SDK_Duration(t *testing.T) {
+	type args struct {
+		crdObj *appmesh.Duration
+		sdkObj *appmeshsdk.Duration
+		scope  conversion.Scope
+	}
+	tests := []struct {
+		name       string
+		args       args
+		wantSDKObj *appmeshsdk.Duration
+		wantErr    error
+	}{
+		{
+			name: "normal case",
+			args: args{
+				crdObj: &appmesh.Duration{
+					Unit:  "ms",
+					Value: int64(200),
```

Comment:
i think this didn't compile.. Unit is a enum and Value is a int64 value instead of pointer in crd

Reply:
my bad, i thought it's aws.Int64.